ui/app/src/state/actions/content.ts

```diff
@@ -83,15 +83,11 @@ export const pasteItemWithPolicyValidation = /*#__PURE__*/ createAction<{ path:
 );
 
 export const unlockItem = /*#__PURE__*/ createAction<{ path: string; notify?: boolean }>('UNLOCK_ITEM');
-export const unlockItemCompleted = /*#__PURE__*/ createAction<{ path: string }>('UNLOCK_ITEM_COMPLETED');
```

Comment:
Because this is an asynchronous action completion, we should probably keep it for having the full cycle of actions (fetch, complete, failure). Even if the reducer doesn't do anything at this point.